ddtrace/contrib/django/patch.py

```diff
@@ -713,49 +708,17 @@ def traced_login(django, pin, func, instance, args, kwargs):
         request = get_argument_value(args, kwargs, 0, "request")
         user = get_argument_value(args, kwargs, 1, "user")
 
-        if not config._appsec_enabled or mode == "disabled":
+        if mode == "disabled":
```

Comment:
wasn't there some confusing logic around when this should or should not be disabled? Are we changing that, or does this somehow just simplify it?

Reply:
Oh you moved the appsec_enabled check into the login and auth methods that get called. Just curious, what's the reason for doing that? Doesn't it make what's actually going to run a bit more obscure?

Reply:
My intention with that change is to remove references to the AppSec product from the contrib code. I'm still getting a sense for exactly how much of that approach is constructive.